### ISSUE TITLE
Use `.bracket-matcher .region` to select highlighted brackets.

### ISSUE THREAD

**Issue by dtigue** [blaqbern/idiomatic-dark-syntax#1]
In index.less: Use `.bracket-matcher .region` to select highlighted brackets.

**Comment by dtigue** [comment]
You might already know about this. In Atom the bottom right hand of the screen has a triangle with an exclamation mark in it, and it says 1 deprecation. I clicked it, and it just says, Use .bracket-matcher .region to select highlighted brackets. There is a Report Issue button, so I figured I would report it just in case no one else had. As a side note, it would be nice if a plugin or something was written to state beside the Report Issue button if it has been reported already. Also, maybe it could give you a button for a +1 or something similar. Maybe even just an extra button that takes you to that "previously" reported error and allow you to check the status and/or add a comment. Just a thought. I would do it, but I've got very limited coding skills. I'm basically only a bash scripter and I dabble a bit with perl and python. Just know enough to help make my Sys Admin job easier, and maybe enough to be dangerous. Anyway, there ya go, problem reported.
